composer.json

```diff
@@ -75,7 +75,7 @@
         "willdurand/hateoas-bundle":            "@stable",
         "winzou/state-machine":                 "~0.1",
         "winzou/state-machine-bundle":          "~0.2",
-        "payum/omnipay-bridge": "0.13.*"
+        "payum/omnipay-bridge":                 "~0.12"
```

Comment:
It's covered by `payum` version, composer locks himself on `0.12` so there is no need to change.